src/app/components/WorkDetail/DefinitionList.jsx

```diff
@@ -52,117 +51,62 @@ export const DefinitionList = ({ eReaderUrl, data }) => {
    * @param {array} entries
    * @return {string|null}
    */
-  const parseEntries = (type, entries) => {
-    let list = [...entries];
-    if (type === 'instances') {
-      list = list.map((item) => {
-        let publisher =
-          item && item.agents && item.agents.find(i => i.roles.indexOf('publisher') > -1);
-        publisher = publisher && publisher.name;
-        return Object.assign({}, item, { publisher });
-      });
-    }
+  const parseEntries = (type, entries, workObj) => {
+    const list = [...entries];
     switch (type) {
       case 'language':
         return (
           <ul>
             {list.map((entity, i) => (
```

Comment:
I typically prefer keys/iterators that are >= 3 characters in length, but that's just my PHP linter talking.